### PR TITLE
[1.3.11] Consolidate 1.3.11 Release Notes

### PR DESCRIPTION
**Pull request by jordarlu**
### Description
Consolidate 1.3.11 Release Notes

### Issues Resolved
N/A

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: release-notes/opensearch-release-notes-1.3.11.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+OpenSearch [Release Notes](https://github.com/opensearch-project/OpenSearch/blob/main/release-notes/opensearch.release-notes-1.3.11.md).
+
+OpenSearch Dashboards [Release Notes](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/release-notes/opensearch-dashboards.release-notes-1.3.11.md).
+
+
+## ENHANCEMENTS
+
+## BUG FIXES
+
```
Comment (peterzhuamazon) [inline release-notes/opensearch-release-notes-1.3.11.md:15]: Thanks @jordarlu please remove the enhancements and bug fixes headings as there are no entries.

Thanks.

**Review by peterzhuamazon — APPROVED** [review]

File: release-notes/opensearch-release-notes-1.3.11.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+## MAINTENANCE
+
+### Opensearch ML Commons
+* Increment version to 1.3.11-SNAPSHOT ([#983](https://github.com/opensearch-project/ml-commons/pull/983))
```
Comment (peterzhuamazon) [inline release-notes/opensearch-release-notes-1.3.11.md:19]: Remove `-SNAPSHOT`.

File: release-notes/opensearch-release-notes-1.3.11.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+* Manual CVE fix ([#1010](https://github.com/opensearch-project/ml-commons/pull/1010))
```
Comment (peterzhuamazon) [inline release-notes/opensearch-release-notes-1.3.11.md:20]: Upgrade guava to 31.0.1 and checkstyle to 8.38.

File: release-notes/opensearch-release-notes-1.3.11.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+### Opensearch Security
+* Update `spring-core` from 5.3.26 to 5.3.27 ([#2873](https://github.com/opensearch-project/security/pull/2873))
+* Add Andrey Pleskach (Willyborankin) to Maintainers ([#2851](https://github.com/opensearch-project/security/pull/2851))
```
Comment (peterzhuamazon) [inline release-notes/opensearch-release-notes-1.3.11.md:24]: Change to `Add Andrey Pleskach (Willyborankin) and Andriy Redko (reta) to Maintainers`.

File: release-notes/opensearch-release-notes-1.3.11.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+### OpenSearch Security Dashboards
+* Use version from package.json for integration tests ([#1446](https://github.com/opensearch-project/security-dashboards-plugin/pull/1446))
```
Comment (peterzhuamazon) [inline release-notes/opensearch-release-notes-1.3.11.md:27]: This should be #1446 not #1466.

File: release-notes/opensearch-release-notes-1.3.11.md (hunk 3 above)
Comment (jordarlu) [reply release-notes/opensearch-release-notes-1.3.11.md]: updated. thanks

File: release-notes/opensearch-release-notes-1.3.11.md (hunk 2 above)
Comment (jordarlu) [reply release-notes/opensearch-release-notes-1.3.11.md]: removed.. thanks

File: release-notes/opensearch-release-notes-1.3.11.md (hunk 1 above)
Comment (jordarlu) [reply release-notes/opensearch-release-notes-1.3.11.md]: got it ... thanks .. removed

**Pull request merged**